index.html

```diff
@@ -0,0 +1,118 @@
+        <p id="dropdown-label">Would you recommend freeCodeCamp to a friend?</p>
+        <label class="flex-block">
+          <input name="user-recommend" value="definitely" type="radio" class="input-radio">
+          Definitely
+        </label>
+        <label class="flex-block">
+          <input name="user-recommend" value="definitely" type="radio" class="input-radio">
+          Maybe
+        </label>
+        <label class="flex-block">
+          <input name="user-recommend" value="definitely" type="radio" class="input-radio">
+          Not sure
+        </label>
+      </div>
+
+      <div class="form-group">
+        <label for="dropdown" id="dropdown-label">What is your favorite feature of freeCodeCamp?
+        </label>
+        <select name="dropdown" id="dropdown">
+          <option value="Select an option" disabled>Select an option</option>
+          <option value="Challanges">Challanges</option>
+          <option value="Projects">Projects</option>
+          <option value="Comunity">Comunity</option>
+          <option value="Open source">Open source</option>
+        </select>
+      </div>
+      <div class="form-group">
+        <p>
+          What would you like to see improved?
+          <span class="clue">(Check all that apply)</span>
+        </p>
+
+        <label class="flex-block"><input name="prefer" value="front-end-projects" type="checkbox">Front-end
+          Projects</label>
+        <label class="flex-block">
+          <input name="prefer" value="back-end-projects" type="checkbox">Back-end
+          Projects</label>
+        <label class="flex-block"><input name="prefer" value="data-visualization" type="checkbox">Data
+          Visualization</label>
+        <label class="flex-block"><input name="prefer" value="challenges" type="checkbox">Challenges</label>
+        <label class="flex-block"><input name="prefer" value="open-source-community" type="checkbox">Open Source
+          Community</label>
+        <label class="flex-block"><input name="prefer" value="gitter-help-rooms" type="checkbox">Gitter help
+          rooms</label>
+        <label class="flex-block"><input name="prefer" value="videos" type="checkbox">Videos</label>
+        <label class="flex-block"><input name="prefer" value="city-meetups" type="checkbox">City Meetups</label>
+        <label class="flex-block"><input name="prefer" value="wiki" type="checkbox">Wiki</label>
+        <label class="flex-block"><input name="prefer" value="forum" type="checkbox">Forum</label>
+        <label class="flex-block"><input name="prefer" value="additional-courses" type="checkbox">Additional
+          Courses</label>
+      </div>
+      <div class="form-group">
+        <p>Any comments or suggestions?</p>
+        <textarea id="comments" class="input-textarea" rows="6" cols="100" name="comment"
+          placeholder="Enter your comment here..." spellcheck="false"></textarea>
+      </div>
+      <div class="form-group">
```

Comment:
🤓⭐ great job implementing the survey form. however, I would recommend you here to use HTML5 semantic `input` tag with the type of `submit` instead of using the `button` tag with the type of `submit`.

Reply:
It would be nice if you have stated the reason you are requesting this change in order for the student to learn from the comment and not just implement it. Also, it makes the correction more appealing.